proposals/list-patterns.md

```diff
@@ -69,7 +69,12 @@ As a result, an error is produced for something like `case [.., p]: case [p]:` b
 
 If a slice subpattern matches a list or a length value, subpatterns are treated as if they were a direct subpattern of the containing list. For instance, `[..[1, 2, 3]]` subsumes a pattern of the form `[1, 2, 3]`.
 
-`Length` or `Count` properties are assumed to always return a non-negative value, if and only if the type is *indexable*. For instance, the pattern `{ Length: -1 }` can never match an array. The behavior of a pattern-matching operation is undefined if this assumption doesn't hold.
+The following assumptions are made on the members being used:
+
+- `Length` or `Count` properties are assumed to always return a non-negative value, if and only if the type is *indexable*. For instance, the pattern `{ Length: -1 }` can never match an array.
```

Comment:
Consider "The property that makes the type countable is assumed to ...".
In the scenario where both `Length` and `Count` are present, only `Length` gets the benefit of the assumption.